packages/jest-cli/src/Runtime/Runtime.js

```diff
@@ -494,12 +538,16 @@ class Runtime {
       this._shouldAutoMock = true;
       return runtime;
     };
-    const unmock = moduleName => {
+    const unmock = (moduleName: string) => {
       const moduleID = this._getNormalizedModuleID(from, moduleName);
       this._explicitShouldMock[moduleID] = false;
       return runtime;
     };
-    const mock = (moduleName, mockFactory, options) => {
+    const mock = (
+      moduleName: string,
+      mockFactory: Object,
+      options: Object
```

Comment:
Can we add a MockOptions type for this with the specific field (I believe `virtual: boolean`?)